.github/workflows/backport.yml

```
@@ -52,17 +71,17 @@ jobs:
+            gh pr comment ${{ env.PR_NUMBER }} --body "Cannot cherrypick due to conflict."
             exit 1
           else
-            echo "Cherry-pick succeeded, pushing and creating PR..."
-            git push origin backport/prod-${{ github.event.pull_request.number }}
-            
+            echo "Cherry-pick OK, pushing..."
+
+        # push
+            git push origin backport/prod-${{ env.PR_NUMBER }}
+        # create PR
+               
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)